core/templates/pages/contributor-dashboard-page/modal-templates/certificate-download-modal.component.spec.ts

```diff
@@ -164,18 +161,50 @@ describe('Contributor Certificate Download Modal Component', () => {
     component.fromDate = tomorrow.toDateString();
     component.toDate = today.toDateString();
 
-    component.downloadCertificate();
+    component.validateDate();
 
     expect(component.errorsFound).toBeTrue();
     expect(component.errorMessage).toEqual('Invalid date range.');
   });
 
+  it('should function correctly', () => {
+    const today = new Date();
+    let fromDate = new Date();
+    let toDate = new Date();
```

Comment:
```suggestion
    const fromDate = new Date();
    const toDate = new Date();
```

You can still use `const` as long as the variable is not reassigned.

Reply:
Done